### ISSUE TITLE
 @RequestPart("files")  Flux<FilePart> upload error  on the swagger-ui

### ISSUE THREAD

**Issue by oursy** [springdoc/springdoc-openapi#678]
Given the following endpoint：
```
  @PostMapping(value = "/files", produces = {MediaType.APPLICATION_JSON_VALUE}, consumes = {MediaType.MULTIPART_FORM_DATA_VALUE})
    @Operation(summary = "files")
    public Flux<Void> handleFileUpload(
            @RequestPart("files") @Parameter(description = "files",
                    content = @Content(mediaType = MediaType.APPLICATION_OCTET_STREAM_VALUE))
                    Flux<FilePart> filePartFux) throws IOException {
        File tmp = File.createTempFile("tmp", "");
        return filePartFux.flatMap(filePart -> {
            Path path = Paths.get(tmp.toString() + filePart.filename());
            System.out.println(path);
            return filePart.transferTo(path);
        });
    }
```
Calling with swaggee-ui returns failure

![image](https://user-images.githubusercontent.com/9456577/82423100-e631c180-9ab5-11ea-8a4b-18157578a79e.png)

I am using springdoc 1.3.9 & spring boot 2.3.0

**Comment by bnasslahsen** [comment]
@oursy, 

 The generated OpenAPI documentation looks correct.
 The problem seems from the request on the swagger-ui. 
 It needs further investigations to be sure if the issue is on springdoc-openapi or on swagger-ui side.

**Comment by bnasslahsen** [comment]
@oursy, 

 I confirm the generated spec is correct, for your sample. 

 There is an open feature related to uploading files from the swagger-ui. Once fixed, we will add reference to the last version of the ui on springdoc-openapi.

There is an open feature request on the swagger-ui project. And here is the link for the right issue to follow:
 - https://github.com/swagger-api/swagger-ui/issues/4600

**Comment by bnasslahsen** [comment]
The fix will be available on v1.4.1.
